organisaatio-ui/src/components/Sivut/Ryhmat/__snapshots__/Ryhmat.test.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Ryhmat Rendering Renders without crashing 1`] = `<Component />`;
```

Comment:
This would make more sense if rendered with some feasible properties

Reply:
I think there is no props for now,if i will remove unused props from the component itself, will it be okay then?